complex objects/cargo/src/lib.rs

```diff
@@ -9,6 +9,7 @@
 // specific language governing permissions and limitations under the License.
 
 extern crate libc;
+extern crate libsqlite3_sys;
```

Comment:
Worth a comment here, I think, 'cos I'm not sure why you need this!

Reply:
Leftovers from earlier exploration. Don't need it. Removing.